crates/rnote-cli/src/cli.rs

```diff
@@ -32,37 +34,66 @@ pub(crate) enum Commands {
-        xopp_dpi: Option<f64>,
+        #[arg(long, default_value_t = XoppImportPrefs::default().dpi)]
+        xopp_dpi: f64,
     },
     /// Exports the Rnote file(s) and saves it in the desired format.{n}
-    /// When using --output-file, only one input file can be given.{n}
-    /// The export format is recognized from the file extension of the output file.{n}
-    /// When using --output-format, the same file name is used with the extension changed.{n}
-    /// --output-file and --output-format are mutually exclusive but one of them is required.{n}
-    /// Currently `.svg`, `.xopp` and `.pdf` are supported.{n}
-    /// Usages: {n}
-    /// rnote-cli export --output-file [filename.(svg|xopp|pdf)] [1 file]{n}
-    /// rnote-cli export --output-format [svg|xopp|pdf] [list of files]
+    /// See subcommands for usage
     Export {
         /// the rnote save file
+        #[arg(global = true)]
         rnote_files: Vec<PathBuf>,
-        /// the export output file. Only allows for one input file. Exclusive with output-format.
-        #[arg(short = 'o', long, conflicts_with("output_format"), required(true))]
-        output_file: Option<PathBuf>,
-        /// the export output format. Exclusive with output-file.
-        #[arg(short = 'f', long, conflicts_with("output_file"), required(true))]
-        output_format: Option<String>,
-        /// export with background
-        #[arg(short = 'b', long)]
-        with_background: Option<bool>,
-        /// export with background pattern
-        #[arg(short = 'p', long)]
-        with_pattern: Option<bool>,
+        /// The action that will be performed if the to be exported file(s) already exist(s)
+        #[arg(long, default_value = "ask", global = true)]
+        on_conflict: OnConflict,
+        /// export without background
+        #[arg(short = 'b', long = "no-background", action = ArgAction::SetFalse, global = true)]
+        background: bool,
+        /// export without background pattern
+        #[arg(short = 'p', long = "no-pattern", action = ArgAction::SetFalse, global = true)]
+        pattern: bool,
+        #[command(subcommand)]
+        export_command: ExportCommands,
     },
 }
 
+#[derive(ValueEnum, Copy, Clone, Debug, Default)]
+pub(crate) enum OnConflict {
+    #[default]
+    /// Ask before Overwriting
+    Ask,
+    /// Overwrite Files
+    Overwrite,
+    #[value(skip)]
+    AlwaysOverwrite,
+    /// Skip current Export
+    Skip,
+    #[value(skip)]
+    AlwaysSkip,
+    /// Add number to the end of the file
+    Suffix,
+    #[value(skip)]
+    AlwaysSuffix,
+}
+
+impl std::fmt::Display for OnConflict {
+    fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
+        write!(
+            f,
+            "{}",
+            match self {
+                Self::Ask => "Open existing file for inspection and ask again",
+                Self::Overwrite => "Overwrite existing file",
+                Self::AlwaysOverwrite => "Always Overwrite exitings file",
```

Comment:
Here are some typos: `existings` -> `existing` and `Overwrite` should be lowercase.